backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/inoutcandidate/api/impl/ShipmentSchedulePA.java

```diff
@@ -433,6 +445,18 @@ public void updatePreparationDate_Override(final Timestamp preparationDate, fina
 		);
 	}
 
+	@Override
+	public void updateExportStatus(final String exportStatus, final PInstanceId pinstanceId)
+	{
+		updateColumnForSelection(
+				I_M_ShipmentSchedule.COLUMNNAME_ExportStatus,
+				exportStatus,
+				false,
+				pinstanceId,
+				true
```

Comment:
```suggestion
				false /* invalidate */
```

i don't see the need to invalidate..why invalidate=true?
PS..proposing change to false..

Reply:
initially thought a cache reset is needed, seems it's not!